### DISCUSSION TITLE
how to modify a theme to specify the color of trait type on rust files

### DISCUSSION THREAD

**Discussion by golovo** [helix-editor/helix#10293] [General]
thanks

**Comment by TornaxO7** [comment]
See https://docs.helix-editor.com/themes.html#scopes for more information.

I haven't done something like that yet, but if I'm understanding it correctly:
1. Create a simple rust file with the relevant part of your code
2. Get the tree-sitter-scope of this code
3. Look under which scope and path your trait pops up
4. Add your changes to the `theme.toml`
